routes/orderRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const orderController = require('../controllers/orderController');
+const { loginRequired } = require('../utils/auth');
+
+router.get('/searching', loginRequired, orderController.getOrderByOrderId);
```

Comment:
RESTful한 엔드포인트를 위해 `orderId`를 path parameter로 받아오는 것으로 수정하면 좋을 것 같네요!
```suggestion
router.get('/:orderId', loginRequired, orderController.getOrderByOrderId);
```